app/design/adminhtml/Magento/backend/Magento_Ui/web/css/source/module/_data-grid.less

```diff
@@ -41,6 +41,11 @@
     overflow-x: auto;
     padding-bottom: 1rem;
     padding-top: @data-grid-header-row__indent;
+    .col-imported_file {
```

Comment:
Hi @suryakant-krish. Thanks for collaboration. Are you sure that hide <p> element is the best way to vertical align text?

Reply:
I have resolved this issue using php and removed my old css.